### PR TITLE
Add Team Progress Checklist email

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: SR2021/2020-12-10-team-progress-checklist.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+There are only 5 weeks until our [first round of league matches](https://studentrobotics.org/events/sr2021/league-1/) so we wanted to check in and make sure you're making good progress! We've put together the below checklist of things that should have happened by now to help you figure out if you're on track for success.
+
+ - Team Leader should be in Discord
+ - Team should be in Discord
+ - (If multiple teams) Competitors have formed separate teams and have the correct team role in Discord
+ - Microgames from Kickstart completed
+ - You have a robot that moves
+ - Everyone in the team has access to the team's code
+ - Everyone in the team has run the code
```
Comment (RealOrangeOne) [inline SR2021/2020-12-10-team-progress-checklist.md:15]: I think we should probably expand these into slightly more full sentences.

```suggestion
 - Team Leaders are be in Discord, and have the "Team Leader" role
 - Competitors are be in Discord
 - (If multiple teams) Competitors have formed separate teams and have the correct team role in Discord
 - Microgames from Kickstart are completed
 - You have a robot that moves, however intelligently
 - Everyone in the team has access to the team's code and can collaborate
 - Everyone in the team has run the code and understands how it works
```

**Comment by RealOrangeOne** [comment]
We should also mention friendly matches

File: SR2021/2020-12-10-team-progress-checklist.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2021/2020-12-10-team-progress-checklist.md:15]: Regarding the last of these -- measuring understanding is potentially fairly hard. I think there's value in the team members all having run the code even if they don't yet understand it. Perhaps break that into two parts?

File: SR2021/2020-12-10-december-update.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ - You should have a robot that moves, with some intelligence
+ - Everyone in the team should have access to the team's code and can collaborate
+ - Everyone in the team should have run the code and understands how it works
+
+## Friendlies
+
+Some of the competitors in Discord have expressed an interest in us running friendly matches. We think this is a great idea, so on the 9th of January we will be running non-scored matches for any team that wishes to take part. More information around how to submit code and opt in to the matches will follow!
+
+
+If you haven't completed any of these items just let us know and we'll do our best to help out!
```
Comment (RealOrangeOne) [inline SR2021/2020-12-10-december-update.md:26]: Should probably reword this so it's less negative.

```suggestion
If you need help with any of these items, just let us know and we'll do our best to help!
```

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**